### PR TITLE
#163683556 consume the sign in route

### PR DESCRIPTION
**Pull request by Chiazokam**
#### What does this PR do?
This PR consumes the sign in route

#### Description of Task to be completed?
- consume the sign in route
- create a helper file to hold all helper
- display errors in a div
- implement the logout function

#### How should this be manually tested?
- Clone the repo - `https://github.com/Chiazokam/Politico`
- Enter into the project directory - `cd Politico`
- switch to this branch - `checkout ft-fetch-signin-163683556`
- Start the server `npm run start:dev`
- Go to the `frontend` folder and run the index.html file on the browser
- Navigate to the sign up page

#### Any background context you want to provide?
Nil

#### What are the relevant pivotal tracker stories?
[#163683556](https://www.pivotaltracker.com/story/show/163683556)

### CODE REVIEW

File: frontend/scripts/fetch/signup.js [hunk 1]
```diff
@@ -58,7 +45,8 @@ function signUp(e) {
     })
     .then((res) => res.json())
     .then((response) => {
-        const { error } = response;
+        const { error, data } = response;
+        console.log(data)
```
Comment (andela-gike) [inline frontend/scripts/fetch/signup.js:49]: Remove the console statement

File: server/controllers/userControllers.js [hunk 2]
```diff
@@ -42,7 +42,8 @@ class UserController {
                     othername: userResponse[0].othername,
                     email: userResponse[0].email,
                     phone: userResponse[0].phone,
-                    passportUrl: userResponse[0].passportUrl,
+                    passportUrl: userResponse[0].passporturl,
+                    isAdmin: userResponse[0].isadmin,
                 };
                 const token = jwt.sign(data, process.env.SECRET_KEY, { expiresIn: '2d' });
```
Comment (andela-gike) [inline server/controllers/userControllers.js:48]: What is contained in your data object, you use to create your jwt token

File: server/controllers/userControllers.js [hunk 3]
```diff
@@ -87,7 +88,8 @@ class UserController {
                     othername: response[0].othername,
                     email: response[0].email,
                     phone: response[0].phone,
-                    passportUrl: response[0].passportUrl,
+                    passportUrl: response[0].passporturl,
+                    isAdmin: response[0].isadmin,
                   };
                 const token = jwt.sign(data, process.env.SECRET_KEY, { expiresIn: '2d' });
```
Comment (andela-gike) [inline server/controllers/userControllers.js:94]: Same here as above

**Review by andela-gike — CHANGES_REQUESTED** [review]

File: frontend/scripts/fetch/signup.js (hunk 1 above)
Comment (Chiazokam) [reply frontend/scripts/fetch/signup.js:49]: Alright

File: server/controllers/userControllers.js (hunk 2 above)
Comment (Chiazokam) [reply server/controllers/userControllers.js:48]: The user's details including the name, email, phone .... and isAdmin status

**Pull request closed without merging**